CONTRIBUTORS.yaml

```diff
@@ -1885,3 +1886,22 @@ biont:
     funding_system: cordis
     funding_statement: |
       Co-funded by the European Union
+
+ukri:
+    name: UKRI
+    github: false
+    joined: 2023-10
+    avatar: "https://github.com/galaxyproject/training-material/blob/main/shared/images/ukri.png?raw=true"
+    url: https://elixiruknode.org/projects/elixir-uk-dash/
+    funder: true
+    funding_statement: |
+      UK Research and Innovation (UKRI) is the national funding agency investing in science and research in the UK. Invest in research that enriches lives, drives economic growth, and creates jobs and high-quality public services.
```

Comment:
Just to check the relationship here, UKRI is a funding body that funds DaSH, correct? If so I think adding just the second entry would make the most sense, as we don't really annotate the funding _agencies_ themselves, but rather the specific grant that funded the work.

```suggestion
```